src/main/java/seedu/duke/ItemList.java

```diff
@@ -0,0 +1,45 @@
+package seedu.duke;
+
+import java.util.List;
+
+public class ItemList {
+    private final Ui ui = new Ui();
+    private final List<Item> items;
```

Comment:
since both user and transaction use ArrayList, we should use ArrayList for this oso